pandas/core/generic.py

```diff
@@ -3650,8 +3649,11 @@ def clip(self, lower=None, upper=None, out=None, axis=None):
         3  0.230930  0.000000
         4  1.100000  0.570967
         """
-        if out is not None:  # pragma: no cover
-            raise Exception('out argument is not supported yet')
+        from pandas import Panel
+        if isinstance(self, Panel):
```

Comment:
You inadvertently stumbled across a bug.  `com.ABCPanel` should apply to both `Panel` and `Panel4D` but it only currently applies to `Panel`.  Will squash with this fix.